packages/destination-actions/src/destinations/personas-messaging-twilio/index.ts

```diff
@@ -43,6 +43,12 @@ const destination: DestinationDefinition<Settings> = {
         description: 'Source ID',
         type: 'string',
         required: true
+      },
+      webhookUrl: {
+        label: 'Webhook URL',
+        description: 'Webhook URL that will receive all events for the sent message',
+        type: 'string',
```

Comment:
You can add 
```suggestion
        type: 'string',
        format: 'uri',
```

If this is expected to be a valid uri.